adbe/adb_enhanced.py

```diff
@@ -1602,3 +1607,119 @@ def print_notifications():
         for action in notification_actions:
             print_message('Action: %s' % action)
         print_message('')
+
+
+# Alarm Enum
+class AlarmEnum(Enum):
+    TOP = 't'
+    PENDING = 'p'
+    HISTORY = 'h'
+    ALL = 'a'
+
+
+# Prettify alarm logs
+def alarm_manager(param):
+    cmd = "dumpsys alarm"
+    c, o, e = execute_adb_shell_command2(cmd)
+
+    if c != 0:
+        print_error_and_exit("Something gone wrong on "
+                             "dumping alarms. Error: %s" % e)
+        return o
+
+    if isinstance(param, AlarmEnum):
```

Comment:
Consider inverting this `if not isinstance(param, AlarmEnum):\n return` to reduce the indentation in the rest of the code